Makefile

```diff
@@ -0,0 +1,43 @@
+#
+#
+#
+
+all:
+	echo No default target - try ve, test, lint, run
+	false
+
+COVER_PERCENT:=60
+
+# FIXME:
+# - depends on python > 3.6 (bad news for RHEL7 /and/ RHEL8)
+# - Add a check for correct version and fail fast
+
+ve: ve/pyvenv.cfg
+ve/pyvenv.cfg: requirements.txt
+	python3 -m venv ve
+	. ve/bin/activate; pip install -r requirements.txt
+
+test: ve
+	. ve/bin/activate; pytest
+
+cover: ve
+	. ve/bin/activate; pytest \
+	    --cov=pyth_observer \
+	    --cov-report=html \
+	    --cov-report=term \
+	    --cov-fail-under=$(COVER_PERCENT)
```

Comment:
imo a min code coverage threshold is going to be an irritant going forward. better to enforce the existence of unit tests via code review than a (likely noisy) automated computation.

Reply:
Yeah - but when I saw that it was currently only 60% covered, I figured I would put a minimum in place....  Hopefully, we can trust that others will run the tests and not propose changes that cause this to be an irritant.  That being said, it is not core to my proposed changes.